src/makeCalculator.js

```diff
@@ -1,10 +1,39 @@
 'use strict';
 
-/**
- * @return {object}
- */
 function makeCalculator() {
-  // write code here
+  // Object that stores the calculator's result and methods
+  const calculator = {
+    // Initial result is 0
+    result: 0,
+
+    // Addition operation
+    add: (num) => calculator.result + num,
+
+    // Subtraction operation
+    subtract: (num) => calculator.result - num,
+
+    // Multiplication operation
+    multiply: (num) => calculator.result * num,
+
+    // Division operation
+    divide: (num) => calculator.result / num,
```

Comment:
[CODE STYLE] - In the `divide` method, use the `/=` operator to update the `result` property. Change `calculator.result / num` to `calculator.result /= num`.